.github/workflows/ocm.yml

```
@@ -4,19 +4,18 @@ name: OCM
-env:
-  GO_VERSION: '1.26'
 
 jobs:
     e2e-test:
         runs-on: ubuntu-latest
         steps:
         - run: echo ${{ github.event.client_payload.ref }}
         - run: echo ${{ github.event.client_payload.sha }}
-        - uses: actions/checkout@v3
+        - name: Check out code
+          uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)